From ff7c13b41512cd9d9b4df88d7cb20aec976f41e7 Mon Sep 17 00:00:00 2001
From: Hong-Mei Li <a21834@motorola.com>
Date: Fri, 25 Sep 2015 23:16:40 -0700
Subject: [PATCH 218/959] IKSWM-4458 motoconfig: Enable multicast to support
 IGMP

Enable multicast and mroute to support IGMP packets and DLNA service.

Change-Id: I7378fac158bde4ace4679410058b305fa519ef6a
Signed-off-by: Hong-Mei Li <a21834@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index 447f9d8dddb..54976981d77 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -38,3 +38,6 @@ CONFIG_LOCKUP_IPI_CALL_WDT=y
 CONFIG_ANDROID_LMK_ADJ_RBTREE=y
 # networking related
 CONFIG_NETFILTER_XT_MATCH_TCPMSS=y
+CONFIG_IP_MULTICAST=y
+CONFIG_IP_MROUTE=y
+CONFIG_IP_PIMSM_V2=y
-- 
2.11.0

